libnavigation-base/src/main/java/com/mapbox/navigation/base/internal/extensions/RouteOptionsUtils.kt

```diff
@@ -1,6 +1,6 @@
 @file:JvmName("MapboxRouteOptionsUtils")
 
-package com.mapbox.navigation.base.extensions
+package com.mapbox.navigation.base.internal.extensions
```

Comment:
should ` fun RouteOptions.Builder.coordinates` be internal?

Reply:
Thought about this too and IMO _everything should be private until proven otherwise_ 😅 In other words, it's better to keep it `internal` and if customers request this we can always make it `public` afterwards (but not the other way around as we'd break SemVer 😉).

Reply:
it cannot be _internal_, `examples` uses it. The same as above